resolwe_bio/tools/bamtobigwig.sh

```diff
@@ -8,6 +8,12 @@ bin_size="$3"
 time_limit="$4"
 species="$5"
 
+BAM_SEQUENCES=`samtools view -c "${bam_file}"`
+if [ ${BAM_SEQUENCES} == 0 ]; then
```

Comment:
We also may want to put `${BAM_SEQUENCES}` into quotation marks in case `samtools` returns something weird.